src/client/menu/videomenu.c

```diff
@@ -503,29 +505,52 @@ VID_MenuInit(void)
 		s_intensity_slider.generic.name = "color intensity";
 		s_intensity_slider.generic.x = 0;
 		s_intensity_slider.generic.y = (y += 10);
-		s_intensity_slider.cvar = "gl3_intensity";
+		s_intensity_slider.cvar = "r_intensity";
 		s_intensity_slider.minvalue = 0.1f;
 		s_intensity_slider.maxvalue = 256.0f;
 
 		s_overbrightbits_slider.generic.type = MTYPE_SLIDER;
 		s_overbrightbits_slider.generic.name = "overbrights";
 		s_overbrightbits_slider.generic.x = 0;
 		s_overbrightbits_slider.generic.y = (y += 10);
-		s_overbrightbits_slider.cvar = "gl3_overbrightbits";
+		s_overbrightbits_slider.cvar = "r_overbrightbits";
 		s_overbrightbits_slider.minvalue = 0.1f;
 		s_overbrightbits_slider.maxvalue = 256.0f;
 
-		gl3_colorlight = Cvar_Get("gl3_colorlight", "1", CVAR_ARCHIVE);
+		r_colorlight = Cvar_Get("r_colorlight", "1", CVAR_ARCHIVE);
 		s_colorlight_list.generic.type = MTYPE_SPINCONTROL;
 		s_colorlight_list.generic.name = "color light";
 		s_colorlight_list.generic.x = 0;
 		s_colorlight_list.generic.y = (y += 10);
 		s_colorlight_list.itemnames = yesno_names;
-		s_colorlight_list.curvalue = (gl3_colorlight->value != 0);
+		s_colorlight_list.curvalue = (r_colorlight->value != 0);
 	}
-	else
+	else if (strcmp(vid_renderer->string, "gl1") == 0)
 	{
-		gl3_colorlight = 0;
+		s_intensity_slider.generic.type = MTYPE_SLIDER;
+		s_intensity_slider.generic.name = "color intensity";
+		s_intensity_slider.generic.x = 0;
+		s_intensity_slider.generic.y = (y += 10);
+		s_intensity_slider.cvar = "r_intensity";
+		s_intensity_slider.minvalue = 1.0f;
+		s_intensity_slider.maxvalue = 10.0f;
+
+		s_overbrightbits_slider.generic.type = MTYPE_SLIDER;
+		s_overbrightbits_slider.generic.name = "overbrights";
+		s_overbrightbits_slider.generic.x = 0;
+		s_overbrightbits_slider.generic.y = (y += 10);
+		s_overbrightbits_slider.cvar = "r_overbrightbits";
+		s_overbrightbits_slider.minvalue = 0;
+		s_overbrightbits_slider.maxvalue = 3;
+		s_overbrightbits_slider.slidestep = 1;
```

Comment:
Using the very same slider structure for both GL1 and GL3 "overbrightbits" produces some undesired behavior: if you begin with GL1 and switch to GL3, the "slidestep" keeps being `1`, so you lose the capability of changing its value by steps of `0.1` until you restart the game.
Also, since "overbright(bits)" is an integer value for GL1, maybe it's convenient to print it without decimals (`.printformat = "%.0f";`).
See the FOV slider for an example.